app/views/courses/vacancies.html.erb

```diff
@@ -1,10 +1,61 @@
 <%= content_for :page_title, "Vacancies" %>
 
-<%= manage_ui_link_to "Back", "/organisation/#{params[:provider_code]}/courses/#{@course[:course_code]}", class: "govuk-back-link" %>
+<%= manage_ui_link_to "Back", "/organisation/#{params[:provider_code]}/courses/#{@course.course_code}", class: "govuk-back-link" %>
```

Comment:
maybe we should create another link helper that calls `manage_ui_link_to`?

Reply:
Not a bad shout 👌